bulletin/index.html

```diff
@@ -34,137 +34,136 @@ <h1 class="logo-brand" itemprop="brand">MACILE</h1>
+			</article>
+			<article class="buarticle buarticle-violet">
+				<img src="/images/bulletin/SA2007/group1-sa2007-11.jpg">
+				<p>Summer Academy 2007. Group of 7th-8th grade students from schools in Itabo, Nigua, El 18, Yogo Yogo, and Haina. Far right, Yahaira Pinales, Dominican teacher that recruited the students. </p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Objectives</h3>
+				<ul class="buarticle-features--vision">
+					<li>Nourture the creativity and curiosity of talented young people.</li>
+					<li>Increase acces to challenging and estimulating learning environments.</li>
+					<li>Advance engineering education in K-12</li>
+					<li>Increase learning and understanding.</li>
+				</ul>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Summer Academy 2008</h3>
+				<p>The progam was adjusted in light of the observations made in 2007.  The duration was extended to five weeks and the curriculum was redesigned to include math, engineering technology, Spanish literacy, and activities such as chess, field trips, and guest speakers.</p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">About the students</h3>
+				<p>Thirty-five students were admitted and completed the program. (MACILE targets talented students in the top 20% of their class.) There was a required orientation and a meeting with the parents .</p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Schedule and evaluation</h3>
+				<p>
+				 The school day was extended from 12:00 PM to 4:00 PM and students were provided with a healthy lunch as well as breakfast. The student's mathematical and linguistic progress was gauged through a math test and an evaluation questionnaire completed at the end of the program.
+				</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/SA2007/itabo-2007-s.jpg">
+				<p>MACILE Summer Academy 2007 and 2008 were conduceted in July at the Itabo Elementary School in Itabo, San Critóbal, Dominican Republic.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/espanol-2008-4.jpg">
+				<p>Spanish teacher, Antonia Asencio, and members of her class, produly displaying resources use to improve their learning.</p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Los indios de Quisqueya</h3>
+				<h4 class="buarticle-subheading">MACILE Spanish currilum</h4>
+				<p>An innovative active approach to deepen understanding of the language through the history and folblore of the Dominican Republic. A book written exclusively for MACILE by Dr. Rosario Montelongo de Swanson.</p>
+				<p>Los indios de Quisqueya, the accompaning history book and foundation of the MACILE curriculum, was donated by the Tavares Foundation. The author, Juan Tómas Tavares Kelnes, permited its use.</p>
```

Comment:
spelling + grammar

* folklore
* written by

Reply:
> Los indios de Quisqueya, the accompaning history book and foundation of the MACILE curriculum, was donated by the Tavares Foundation. The author, Juan Tómas Tavares Kelnes, permited its use.

Los Indios de Quisqueya and the accompanying history book form the foundation of the MACILE (history or Spanish) curriculum. We thank the Tavares Foundation for the generous donation and the author Juan Tómas Tavares Kelnes for the permission to use his work. 

- I recommend that you qualify curriculum otherwise it reads that these books form the foundation of the entire MACILE curriculum. 
- If you don't want to use the re-write, there are various spelling errors in the original text

Reply:
@cosola Whoops, I think you forgot to include these corrections.